app/Domain/Integrations/Repositories/EloquentIntegrationRepository.php

```diff
@@ -60,7 +60,9 @@ public function update(Integration $integration): void
     public function getDraftsByTypeAndOlderThenMonthsAgo(IntegrationType $type, int $months): Collection
     {
         return IntegrationModel::query()
+            ->distinct()
             ->where('status', 'draft')
+            ->where('type', $type->value)
```

Comment:
Weird that this was not covered with a unit test, should the unit test be updated and include different integration types? I think the test "Should not be selected: wrong type" is still wrong because it is an active integration

Reply:
Yes, the status in the test was the problem. It has been updated.